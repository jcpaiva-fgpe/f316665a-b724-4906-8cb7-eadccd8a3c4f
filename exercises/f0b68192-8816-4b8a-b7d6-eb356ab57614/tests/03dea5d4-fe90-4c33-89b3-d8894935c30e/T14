25 28
1111110001011100000111111111
1111110010011110111111111111
1111110010000110111111111111
1111110000011000000111111111
1111110110011110011111111111
1111110010011110011111111111
1111110100011101011111111111
1111110010011011011111111111
1111110110011110001111111111
1111110000011111011111111111
1111110110011110111111111111
1111110000011111101111111111
1111110110011111011111111111
1111111100011110111111111111
1111111011011111001111111111
1111111101011110011111111111
1111111001010101111111111111
1111111111000101001111111111
1111111001010000111111111111
1111111110000101101111111111
1111111101000101111111111111
1111111011101101111111111111
0000001111111111000000000000
0000001111111111000000000000
0000001111111111000000000000
