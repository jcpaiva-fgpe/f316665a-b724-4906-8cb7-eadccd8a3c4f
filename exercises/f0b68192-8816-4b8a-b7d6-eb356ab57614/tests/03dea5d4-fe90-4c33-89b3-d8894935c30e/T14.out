1111110110011111011111111111
